CONTRIBUTING.md

```diff
@@ -57,25 +57,37 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://git.k8s.io/community/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Releases
+
+Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track releases.
+
+- Minor versions are planned and scheduled twice / year.
+- Patch versions can be planned and scheduled each month.
+- Code freeze is in effect 72 hours (3 days) before a release.
+  - Exceptions might be filed from time to time to address critical bug fixes or issues.
```

Comment:
I think we might need more details here
- Only critical bug fixes may be merged in between freeze & release
- Have to be approved by at least n (2?) project maintainers
- ... ?

Reply:
done ptal